spec/syntax.md

```diff
@@ -481,19 +567,32 @@ text-char = %x0-5B         ; omit \
 
 ### Literals
 
-**_Literal_** is used for matching variants and providing input to _expressions_.
-
-**_Quoted_** literals may include content with any Unicode code point,
+A **_literal_** is a character sequence that appears outside
+of _text_ in various parts of a _message_.
```

Comment:
This phrasing seems backwards, since _text_ can only appear at the top level of a pattern and _literal_s can appear in many places, but I'm not sure how to rephrase it.